modules/luajit/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "homepage": "https://luajit.org/index.html",
+    "maintainers": [
+        {
+            "email": "26427366+UebelAndre@users.noreply.github.com",
+            "github": "UebelAndre",
+            "github_user_id": 26427366,
+            "name": "UebelAndre"
+        }
+    ],
+    "repository": [
+        "github:LuaJIT/LuaJIT"
+    ],
+    "versions": [
+        "2.0.5",
+        "2.1.0-beta3"
```

Comment:
is this an official version? if not, we should try to name it according to Bazel module version semantics, which would be `2.1.0-beta.3` (`2.1.0-beta3` would be considered older than `2.1.0-beta10`)